pyhub/mcptools/microsoft/excel/com/decorators.py

```
@@ -0,0 +1,184 @@
+def ensure_com_thread(func: Callable[..., T]) -> Callable[..., T]:
+    """
+    Decorator to ensure COM is initialized for the current thread
+    """
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs) -> T:
+        com_initialized = False
+        try:
+            # Check if COM is already initialized
+            pythoncom.CoInitialize()
+            com_initialized = True
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)